tsconfig.json

```diff
@@ -3,14 +3,14 @@
     "esModuleInterop": true,
     "exactOptionalPropertyTypes": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["ES2020"],
-    "module": "CommonJS",
-    "moduleResolution": "node",
+    "lib": ["ES2023"],
+    "module": "Node16",
+    "moduleResolution": "Node16",
     "noEmit": true,
     "noErrorTruncation": true,
     "noUncheckedIndexedAccess": true,
     "strict": true,
-    "target": "es2020"
+    "target": "ES2022"
```

Comment:
Following MetaMask/metamask-extension#32529, our targeted browsers fully support ES2022.

Reply:
Our browsers do, but the extension build system does not. We will need to transpile anything that uses the spread operator because Browserify will explode when it tries to parse it.

Spread was introduced in ES2018 though, so we've already crossed that bridge it seems.